src/tests/config/global_configs.py

```diff
@@ -16,6 +16,11 @@
 global_variables = DefaultVariables()
 
 
+def reset_global_variables():
```

Comment:
Seems like we're doing something wrong here, rather than instantiating it somewhere top-level and propagating it anywhere needed 
Can we handle it some other way?

Reply:
We are, but we can't change `global_variables` behavior.
Our assumption was that the container is executed once per set of tests with n environment variables.
In this case we are triggering the test directly from code and not from pytest-cli-command, so when the test-infra CLI is triggers the test, it will not update the `global_variables` with the actual env vars values.